opentreemap/treemap/templates/treemap/partials/search_location_polygon_editing.html

```diff
@@ -0,0 +1,36 @@
+{% extends "treemap/partials/search_location.html" %}
+{% load i18n %}
+
+{% block well-controls %}
+    <a class="draw-area-button" title="{% trans "Draw area on map" %}">
+        <i class="icon-pencil"></i>
+    </a>
+{% endblock %}
+
+{% block alternate-content %}
+    <div class="location-search-alternate" id="draw-area-controls">
+        <a class="cancel-draw-area">
+            <i class="icon-cancel"></i>
+        </a>
+        <span class="text">{% trans "Draw area on map" %}</span>
+    </div>
+    <div class="location-search-alternate" id="custom-area-controls">
+        <a class="clear-location-search">
+            <i class="icon-cancel"></i>
+        </a>
+        <span class="text" style="font-style:italic; font-weight:bold">
```

Comment:
My personal belief is that small one-off styles are more easily managed in the HTML than making you go dig around in the SASS.